spec/controllers/idv/address_controller_spec.rb

```diff
@@ -110,5 +126,12 @@
         expect(response.body).to include(t('idv.errors.pattern_mismatch.zipcode'))
       end
     end
+
+    xit 'has the correct `address_edited` value when submitted twice with the same data' do
```

Comment:
should this be enabled

Reply:
If this is enabled it fails because there is currently a bug. I fixed the bug by not overwriting the address in `pii_from_doc`. That does not happen until #10393 where I enabled this test.